charts/vector/values.yaml

```diff
@@ -340,6 +370,9 @@ persistence:
   selectors: {}
 
   hostPath:
+    # persistence.hostPath.enabled -- If true, use hostPath persistence. Valid for the "Agent" role, if it's disabled
+    # the "Agent" role will use emptyDir for persistence.
```

Comment:
An `emptyDir` is specifically _not_ persisted.

Reply:
yes my mistake! this part should be fixed

Reply:
fixed